example/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "webpack-rails-example",
   "version": "0.0.1",
   "license": "MIT",
+  "scripts": {
+    "compile":  "webpack -d --config config/webpack.config.js",
```

Comment:
I believe you have used `d` here but you wanted `p` and the same on the line below.